crates/topos-tce-storage/src/fullnode/mod.rs

```diff
@@ -318,23 +312,32 @@ impl ReadStore for FullNodeStore {
 
     fn get_target_stream_certificates_from_position(
         &self,
-        position: TargetStreamPositionKey,
+        position: CertificateTargetStreamPosition,
         limit: usize,
     ) -> Result<Vec<(CertificateDelivered, CertificateTargetStreamPosition)>, StorageError> {
-        let starting_position = position.2;
+        let starting_position = position.position;
+        warn!("Starting position {:?}", starting_position);
+        let prefix = TargetSourceListKey(position.target_subnet_id, position.source_subnet_id);
+        warn!("Prefix is: {:?}", prefix);
+        warn!(
+            "target_stream: {:?}",
+            self.index_tables.target_streams.iter()?.collect::<Vec<_>>()
```

Comment:
What do you think of being defensive here and cap the max number of streams we print here? Like `self.index_tables.target_streams.iter()?.take(100).collect::<Vec<_>>()`?

Also: you sure these should be warnings? Look more like `debug!` or `info!` to me.

Reply:
I may just remove those logs I think